data/vibe/data/json.d

```diff
@@ -1141,27 +1141,27 @@ struct Json {
 		return ret.data;
 	}
 	/// ditto
-	void toString(scope void delegate(const(char)[]) @safe sink, FormatSpec!char fmt)
+	void toString(scope void delegate(scope const(char)[]) @safe sink, FormatSpec!char fmt)
 	@trusted {
 		// DMD BUG: this should actually be all @safe, but for some reason
 		// @safe inference for writeJsonString doesn't work.
 		static struct DummyRangeS {
-			void delegate(const(char)[]) @safe sink;
-			void put(const(char)[] str) @safe { sink(str); }
+			void delegate(scope const(char)[]) @safe sink;
+			void put(scope const(char)[] str) @safe { sink(str); }
 			void put(char ch) @trusted { sink((&ch)[0 .. 1]); }
 		}
 		auto r = DummyRangeS(sink);
 		writeJsonString(r, this);
 	}
 	/// ditto
-	void toString(scope void delegate(const(char)[]) @system sink, FormatSpec!char fmt)
+	void toString(scope void delegate(scope const(char)[]) @system sink, FormatSpec!char fmt)
```

Comment:
That's a breaking change isn't it ? Giving it a non-scope delegate will break.

Reply:
True, I'll add a separate overload. Not sure what the warning here was anyway, at least the method should be `@system`.